### ISSUE TITLE
Maintenance mode option in SCMU throws exception

### ISSUE THREAD

**Issue by gbiellem** [Particular/ServiceControl#1166]
## Who's affected

Customers using ServiceControl 2.0

## Symptoms

- With a fresh install of V2.0
- Added  a new instance 
- Click the spanner icon in the instance details
- Click the start maintenance mode option.  

Following exception is shown:

```
System.Configuration.ConfigurationErrorsException: The configuration file has been changed by another program. (C:\Program Files (x86)\Particular Software\Particular.ServiceControl\ServiceControl.exe.config)
   at System.Configuration.MgmtConfigurationRecord.SaveAs(String filename, ConfigurationSaveMode saveMode, Boolean forceUpdateAll)
   at ServiceControlInstaller.Engine.Configuration.ServiceControl.AppConfig.EnableMaintenanceMode() in C:\BuildAgent\work\7189a56f9f44affc\src\ServiceControlInstaller.Engine\Configuration\ServiceControl\AppConfig.cs:line 32
   at ServiceControl.Config.UI.AdvancedOptions.ServiceControlAdvancedViewModel.<>c__DisplayClass34_0.<StartServiceInMaintenanceMode>b__0() in C:\BuildAgent\work\7189a56f9f44affc\src\ServiceControl.Config\UI\AdvancedOptions\ServiceControlAdvancedViewModel.cs:line 75
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ServiceControl.Config.UI.AdvancedOptions.ServiceControlAdvancedViewModel.<StartServiceInMaintenanceMode>d__34.MoveNext() in C:\BuildAgent\work\7189a56f9f44affc\src\ServiceControl.Config\UI\AdvancedOptions\ServiceControlAdvancedViewModel.cs:line 79
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ServiceControl.Config.Commands.StartServiceControlInMaintenanceModeCommand.<ExecuteAsync>d__3.MoveNext() in C:\BuildAgent\work\7189a56f9f44affc\src\ServiceControl.Config\Commands\StartServiceControlInMaintenanceModeCommand.cs:line 45
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at ServiceControl.Config.UI.AdvancedOptions.ServiceControlAdvancedViewModel.<>c__DisplayClass0_0.<<-ctor>b__0>d.MoveNext() in C:\BuildAgent\work\7189a56f9f44affc\src\ServiceControl.Config\UI\AdvancedOptions\ServiceControlAdvancedViewModel.cs:line 27


```

**Comment by danielmarbach** [comment]
Fixed with #1169. Will be released as part of 2.0.1

**Comment by danielmarbach** [comment]
@gbiellem it is [out](https://github.com/Particular/ServiceControl/releases/tag/2.0.1), group announcement will follow soon
